### PR TITLE
SEO-183594-.NET-MAUI-DataGrid-Interlinking

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @Girija-Rajendran 

Please review and merge the PRs,
|Title|Description|
|--|--|
|Category|Money Keywords - .NET MAUI DataGrid|
|Master PR|Not yet created|
|Dev PR| https://github.com/syncfusion-content/featuretour-maui/pull/1127|
|Task|https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/183594/|
|Content Ticket/ Screenshot|image|
|Ticket task| https://internalsupport.bolddesk.com/support/tickets/90121|
|Reference|https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-DataGrid---Finalized-Money-Keywords-Status.aspx|

|Changes made|Reason for changes|
|--|--|
|Renamed image alt text and image namesimage| To understand the content of images, which helps in indexing them correctly|

### CODE REVIEW

File: MAUI/DataGrid/Getting-started.md [hunk 1]
```diff
@@ -359,6 +359,6 @@ public MainPage()
 {% endhighlight %}
 {% endtabs %}
 
-N> Set `HorizontalOptions` and `VerticalOptions` to grid accordingly. When the `SfDataGrid` doesn't obtain finite size from its parent to layout in the View, the predefined `MinimumHeightRequest` and `MinimumWidthRequest`, which is 300, will be acquired.
+N> Set `HorizontalOptions` and `VerticalOptions` to grid accordingly. When the `SfDataGrid` doesn't obtain finite size from its parent to layout in the .NET MAUI DataGrid View, the predefined `MinimumHeightRequest` and `MinimumWidthRequest`, which is 300, will be acquired.
```
Comment (Girija-Rajendran) [inline MAUI/DataGrid/Getting-started.md:362]: revert this

File: MAUI/DataGrid/grid-events.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ The `Datagrid` provides the following events for handling interactions with the
 * [CellRightTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellRightTapped) : Called when a right-click mouse gesture has been recognized on a cell.
 
 ### CellTapped event
-This event will be triggered while tapping a cell in the DataGrid. This event has the [DataGridCellTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellTappedEventArgs.html) as arguments.
+This event will be triggered while tapping a cell in the [.NET MAUI DataGrid](https://www.syncfusion.com/maui-controls/maui-datagrid). This event has the [DataGridCellTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellTappedEventArgs.html) as arguments.
```
Comment (Girija-Rajendran) [inline MAUI/DataGrid/grid-events.md:23]: revert this

File: MAUI/DataGrid/Getting-started.md (hunk 1 above)
Comment (Scholar-jaruha) [reply MAUI/DataGrid/Getting-started.md]: Reverted the changes

**Comment by Girija-Rajendran** [comment]
Hi @Scholar-jaruha,

These details are not correct. Update screenshots in their corresponding rows.
image

File: MAUI/DataGrid/grid-events.md (hunk 2 above)
Comment (Scholar-jaruha) [reply MAUI/DataGrid/grid-events.md]: Reverted the changes.

**Review by Girija-Rajendran — APPROVED** [review]
fine

**Pull request merged**